purchases/src/test/java/com/revenuecat/purchases/common/diagnostics/DiagnosticsTrackerTest.kt

```diff
@@ -38,13 +45,29 @@ class DiagnosticsTrackerTest {
 
     private lateinit var diagnosticsTracker: DiagnosticsTracker
 
+    private lateinit var context: Context
+    private lateinit var appConfig: AppConfig
+
     @Before
     fun setup() {
+        mockkStatic("com.revenuecat.purchases.common.UtilsKt")
```

Comment:
Took me a second to realize I needed this. 

I checked and we don't seem to be calling `unmockkStatic` or `unmockkAll` anywhere to reset these mocks, is this something we should consider adding? Could be a cause of flakiness at some point.